packages/ui/src/views/repo/repo-commits/components/filter.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react'
+
+import { ListActions, SearchBox } from '@/components'
+import { isEmpty } from 'lodash-es'
+
+interface FilterProps {
+  showSort?: boolean
+  sortOptions?: ListActions.DropdownItemProps[]
+  showSearch?: boolean
+  sort: string | null
+  setSort: (sort: string) => void
+  query: string | null
+  setQuery: (query: string) => void
+}
+
+const Filter = ({ showSort, sortOptions, showSearch = true, sort, setSort, query, setQuery }: FilterProps) => {
```

Comment:
do we need filters components in repo-commits? I don't think we actually support any of this on the commit list view?

Reply:
Removed it for now.